src/lib.rs

```diff
@@ -25,6 +25,10 @@
 //!
 //! [`TypeUuid`]: ./trait.TypeUuid.html
 
+pub fn i_have_added_a_new_function() {
+    println!("cool");
```

Comment:
I think we can do better than just "cool" 😉 

```suggestion
    println!("Super Cool!!!!");
```